modules/core/02-client/keeper/keeper_test.go

```diff
@@ -161,7 +161,13 @@ func (suite *KeeperTestSuite) TestSetCreator() {
 	suite.Require().Equal(expectedCreator, retrievedCreator, "Creator is not retrieved correctly")
 
 	// Verify non stored creator is not found
-	retrievedCreator, found = suite.keeper.GetCreator(suite.ctx, "client-0")
+	retrievedCreator, found = suite.keeper.GetCreator(suite.ctx, ibctesting.SecondClientID)
+	suite.Require().False(found, "GetCreator unexpectedly returned a creator")
+	suite.Require().Equal(retrievedCreator, "", "Creator is not empty")
```

Comment:
[nit] you could use suite.Require().Empty() instead (https://pkg.go.dev/github.com/stretchr/testify/assert#Empty)